src/test/resources/META-INF/native-image/org.xerial/sqlite-jdbc/reflect-config.json

```diff
@@ -0,0 +1,15 @@
+[
```

Comment:
This config is only added to the test-suite, so it is not provided to the end-user.
It is only required when using rowsets, which is not code maintained by sqlite-jdbc so it's not up to us to provide the user with this config. It would also cause a direct dependency to java internals which may not be compatible over all jdks/versions.

Same thing for the `resource-config.json` next to this file.